solanches/rest.py

```diff
@@ -140,3 +115,27 @@ def get_cadapio(comercio_nome):
         raise
 
     return jsonify(cardapio), 200
+
+@app.route("/comercio/<comercio_nome>/produto", methods=['POST'])
+def cadastra_produto(comercio_nome):
+    req = request.get_json()
+    
+    assert req, "Erro: json inválido!"
+    assert "nome" in req, "Erro: nome não informado!"
+    assert "descricao" in req, "Erro: descricao não informada!"
+    assert "imagem" in req, "Erro: imagem não informada!"
+    assert "preco" in req, "Erro: preco não informado!"
+    assert "categoria" in req, "Erro: categoria não informada!"
+
+    nome = req.get("nome")
+    descricao = req.get("descricao")
+    imagem = req.get("imagem")
+    preco = req.get("preco")
+    categoria = req.get("categoria")
+
+    try:
+        produto_id = controller.cadastra_produto(nome, descricao, imagem, preco, categoria, comercio_nome)
+    except:
+        raise
+
+    return jsonify(produto_id), 201
```

Comment:
Como tu vai fazer com que o id do produto seja retornado, para não ficar algo solto, tu pode fazer algo assim:
```suggestion
    response = {"message": f"Produto com id {produto_id} criado!"}
    return jsonify(response), 201
```